helloworld-jest-test-sample/.vscode/launch.json

```diff
@@ -0,0 +1,90 @@
+// A launch configuration that compiles the extension and then opens it inside a new window
+// Use IntelliSense to learn about possible attributes.
+// Hover to view descriptions of existing attributes.
```

Comment:
a

Reply:
This comment was a copy/paste from the other helloworld examples, and it reads just fine to me.